tensorflow/python/ops/lookup_ops.py

```diff
@@ -239,16 +239,18 @@ def initializer(self):
 
 @tf_export("lookup.StaticHashTable", v1=[])
 class StaticHashTable(InitializableLookupTableBase):
-  """A generic hash table implementation.
+  """A generic hash table implementation. The table is immutable once
```

Comment:
Linter will complain that the summary needs to be one line ending with a period.

Reply:
Thanks for the heads-up! I've pushed a fix to my branch.